src/git/commit.js

```diff
@@ -50,7 +50,8 @@ function commit (repoPath, message, options, done) {
       }
     });
   } else {
-    const commitFilePath = path.join(repoPath, '/.git/COMMIT_EDITMSG');
+    const gitDirPath = execSync('git rev-parse --absolute-git-dir', { encoding: 'utf8' }).trim();
```

Comment:
This should be run in the `repoPath`, right?

```suggestion
    const gitDirPath = execSync('git rev-parse --absolute-git-dir', { encoding: 'utf8', cwd: repoPath }).trim();
```

Reply:
Yes, it should – Good catch. I will make the change (test needs to be changed, too) and report back. :)

Reply:
I applied the suggested change and it made any changes to the tests unnecessary. If I would have been aware that the tests create their own git environment I wouldn't have created the stub in the first place, sorry, my bad. 😄 I (force-)pushed the changes – Thanks again for catching this!